app/src/main/java/org/wikipedia/readinglist/ReadingListsFragment.kt

```diff
@@ -653,6 +849,16 @@ class ReadingListsFragment : Fragment(), SortReadingListsDialog.Callback, Readin
         }
     }
 
+    fun onListsImportResult(uri: Uri) {
```

Comment:
Would it be possible to show a progress bar or the refresh animation while importing? (and probably the "new" indicator on the imported item).

And I also noticed that if we import the list that has the same name as the existing list, the action will not be able to complete.